plugins/inputs/influxdb/influxdb.go

```diff
@@ -202,6 +203,7 @@ func (i *InfluxDB) gatherURL(
 						"next_gc":         m.NextGC,
 						"last_gc":         m.LastGC,
 						"pause_total_ns":  m.PauseTotalNs,
+						"pause_ns":        m.PauseNs[(m.NumGC+255)%256],
```

Comment:
The index doesn't matter when NumGC == 0 since every element in PauseNs[0:256] will be zero.